Makefile

```diff
@@ -10,7 +10,7 @@ clean:
 	pre-commit clean || true
 
 install-pip-setuptools:
-	python -m pip install -U "pip>=21.2" "setuptools>=38.0" wheel
+	pip install -U "pip>=21.2" "setuptools>=38.0" wheel
```

Comment:
This wasn't wrong before, but it was a bit confusing since our CircleCI conda environment is setup correctly so that `pip` and `python` are in the same environment, and we don't do `python -m` for other commands in the CircleCI setup.